### PR TITLE
Add self_printing_program.py

### PR DESCRIPTION
**Pull request by mmdbalkhi**
#13

### CODE REVIEW

**Review by thevirtualbuddy — APPROVED** [review]
Doesn't follow the rule for Quine. It should not print it's source code by finding from the location of its source code storage.

File: Python/self_printing_program/self_printing_program.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# A python program that prints its source code.
+print(open(__file__).read())
```
Comment (AmirNaghibi) [inline Python/self_printing_program/self_printing_program.py:2]: A program can be called Quine only if it **takes no input at all**. In this case, the program reads its source code as a file, thus making it using itself as input. [src](https://towardsdatascience.com/how-to-write-your-first-quine-program-947f2b7e4a6f)

[Wikipedia](https://en.wikipedia.org/wiki/Quine_(computing)#:~:text=A%20quine%20is%20a%20computer,%22self%2Dcopying%20programs%22.)

Comment (thevirtualbuddy) [reply Python/self_printing_program/self_printing_program.py:2]: Absolutely correct AmirNaghibi. The issue remains open for Quine in python..

**Comment by mmdbalkhi** [comment]
> Doesn't follow the rule for Quine. It should not print it's source code by finding from the location of its source code storage.

But well, I wrote exactly the same thing that was written for. Sorry I did something wrong again

**Comment by thevirtualbuddy** [comment]
There is nothing to be sorry about @mmdbalkhi :)
Yes, your program does fulfill the job to print itself; therefore I merged it. But since it doesn't satisfy the criteria for a Quine program, I have kept the issue remain open.

**Comment by mmdbalkhi** [comment]
> There is nothing to be sorry about @mmdbalkhi :) Yes, your program does fulfill the job to print itself; therefore I merged it. But since it doesn't satisfy the criteria for a Quine program, I have kept the issue remain open.

I think this will happen again! 
‍‍```py
exec(s:='print("exec(s:=%r)"%s)')
```
> Of course, I ask here so that I do not waste your time too much.

I just realized that these kinds of programs should not get input, right? In that case, this will also be the get of the entry, but  Wikipedia approved is this

**Comment by AmirNaghibi** [comment]
I have a PR for this as well in case if you want to review 🙏🏼 

https://github.com/craplab/WeirdScripts/pull/28

**Comment by thevirtualbuddy** [comment]
Reviewed and merged @AmirNaghibi  🥇

**Comment by thevirtualbuddy** [comment]
Yes @mmdbalkhi you're right. But I'm sure it was a good exercise :)

**Pull request merged**